src/main/java/it/gov/pagopa/payhub/activities/service/classifications/trclassifiers/RtNoIufClassifier.java

```diff
@@ -12,14 +12,14 @@
 public class RtNoIufClassifier implements TransferClassifier {
 
 	@Override
-	public ClassificationsEnum classify(TransferDTO transferDTO, PaymentsReportingDTO paymentsReportingDTO, TreasuryDTO treasuryDTO) {
+	public ClassificationsEnum classify(TransferDTO transferDTO, PaymentsReportingDTO paymentsReportingDTO, Treasury treasuryDTO) {
 		if(transferDTO != null && paymentsReportingDTO == null && treasuryDTO == null) {
 			return ClassificationsEnum.RT_NO_IUF;
 		}
 		if (transferDTO != null && paymentsReportingDTO != null && treasuryDTO == null && !getAmountCents(transferDTO).equals(getAmountCents(paymentsReportingDTO))) {
 			return ClassificationsEnum.RT_NO_IUF;
 		}
-		if (transferDTO != null && treasuryDTO != null && paymentsReportingDTO == null && !getAmountCents(transferDTO).equals(getAmountCents(treasuryDTO))) {
+		if (transferDTO != null && treasuryDTO != null && paymentsReportingDTO == null && !getAmountCents(transferDTO).equals(treasuryDTO.getBillAmountCents())) {
```

Comment:
the amount field retrieve logic should still be centralized as done per other entities
```suggestion
		if (transferDTO != null && treasuryDTO != null && paymentsReportingDTO == null && !getAmountCents(transferDTO).equals(getAmountCents(treasuryDTO))) {
```